app/javascript/src/components/Notes/Pane/Form.jsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+
+import { Form, Formik } from "formik";
+import { Check } from "neetoicons";
+import { Pane, Toastr } from "neetoui";
+import { Button, Input, Select } from "neetoui/formik";
+
+import {
+  NOTES_FORM_CONTACT_DATA,
+  NOTES_FORM_INITIAL_VALUES,
+  NOTES_FORM_TAG_DATA,
+  NOTES_FORM_VALIDATION_SCHEMA,
+} from "components/Notes/constants";
+
+const CreateForm = ({ setShowPane }) => (
```

Comment:
The file is named `Form`, but the component is named `CreateForm`. Why?

Reply:
Saw a similiar structure in wheel repository , where there is Notes/Pane/Form.jsx
inside the component is called NoteForm
shall I rename the file to CreateForm.jsx ?

Reply:
Just name the file and component to `Form.jsx`. Also raise an issue in wheel and mention me in it.

Reply:
Formik Form is also used in that component , wouldn't that cause a conflict?

Reply:
@sayoojtmc Go through https://academy.bigbinary.com/learn-react/react-best-practices/naming-conventions-in-react very carefully and get back to me on how we can resolve the issue that you've mentioned.

Reply:
i can import formik Form as
import { Form as FormikForm } from 'formik';
is this the correct way to solve this?

Reply:
Yes.